blueprint/api.apib

```diff
@@ -1358,6 +1358,153 @@ The Stripe plan resource allows users to select a monthly donation amount for a
 
     + Attributes (Record Not Found Response)
 
+# Group Stripe Subscriptions
+
+This endpoint is for creating, editing and returning Stripe Subscriptions.
+
+Stripe Subscriptions allow us to charge a customer's card on a recurring basis. A subscription ties a customer to a particular plan.
+
+## Stripe Subscriptions [/stripe-subscriptions]
+
+### Create a Stripe subscription [POST]
+
++ Parameters
+
+    + id (number) - Id of a Stripe subscription.
+
++ Request
+
+    + Headers
+
+            Accept: application/vnd.api+json
+            Authorization: Bearer {token}
+
++ Response 201 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (Stripe Subscription Response)
+
++ Response 401 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (JSON Web Token Invalid Response)
+
++ Response 403 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (Forbidden Response)
+
++ Response 422 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (Unprocessable Entity Response)
+
+### List all Stripe subscriptions [GET]
+
++ Request
+
+    + Headers
+
+            Accept: application/vnd.api+json
+            Authorization: Bearer {token}
+
+
++ Response 200 (application/vnd.api+json; charset=utf-8)
+
++ Attributes (Stripe Subscriptions Response)
+
++ Response 401 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (JSON Web Token Invalid Response)
+
++ Response 403 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (Forbidden Response)
```

Comment:
I would remove the list all per some recent conversation about just having filters instead.